woocommerce/woocommerce-android
# [UOE] Track events for order management

Part of: #6711

### Description
This PR tracks the following events:

- order_product_quantity_change
- order_product_remove
- order_fee_remove
- order_shipping_method_remove

### Testing instructions

1. Run CreationFocusedOrderCreateEditViewModelTest unit tests
2. Run EditFocusedOrderCreateEditViewModelTest unit tests
3. Go to the Orders -> open an order -> press Edit
       Run the `Check events instructions steps` below and in each case check that the   property `Properties:{"flow":"editing",...}` sends `editing` as the flow property
5. Go to the Orders -> press (+) button -> press Create order
       Run the `Check events instructions steps` below and in each case check that the   property `Properties:{"flow":"creation",...}` sends `creation` as the flow property

#### Check events instructions steps
- Add a product to the order
- Press the + button to increase the product quantity
- Check `🔵 Tracked: order_product_quantity_change, Properties:{"flow":"....",...}` is triggered
- Press the + button to increase the product quantity
- Press the - button to decrease the product quantity
- Check `🔵 Tracked: order_product_quantity_change, Properties:{"flow":"....",...}` is triggered
- Press the - button to decrease the product quantity
- Check `🔵 Tracked: order_product_quantity_change, Properties:{"flow":"....",...}` is NOT triggered because as the product quantity is 1 the app shows the product detail view to remove the product from there
- Press Remove product
- Check `🔵 Tracked: order_product_remove, Properties: {"flow":"....",..}` is triggered
- Add a shipping
- Select the shipping added -> Press Remove shipping
- Check `🔵 Tracked: order_shipping_method_remove, Properties: {"flow":"....",...}` is triggered
- Add a fee
- Select the fee added -> Press Remove fee
- Check `🔵 Tracked: order_fee_remove, Properties: {"flow":".....",...}` is triggered
